components/login/Page.vue

```diff
@@ -11,30 +11,16 @@
           <div class="form-group">
             <div class="input-wrapper">
               <label :class="{ active: email.length > 0 || isEmailFocused }" for="email">Silliman Email</label>
-              <input
-                type="email"
-                id="email"
-                v-model="email"
-                class="form-control"
-                @focus="isEmailFocused = true"
-                @blur="isEmailFocused = email.length > 0"
-                required
-              />
+              <input type="email" id="email" v-model="email" class="form-control" @focus="isEmailFocused = true"
+                @blur="isEmailFocused = email.length > 0" required />
             </div>
           </div>
 
           <div class="form-group">
             <div class="input-wrapper">
               <label :class="{ active: password.length > 0 || isPasswordFocused }" for="password">Password</label>
-              <input
-                :type="passwordInputType"
-                id="password"
-                v-model="password"
-                class="form-control"
-                @focus="isPasswordFocused = true"
-                @blur="isPasswordFocused = password.length > 0"
-                required
-              />
+              <input :type="passwordInputType" id="password" v-model="password" class="form-control"
+                @focus="isPasswordFocused = true" @blur="isPasswordFocused = password.length > 0" required />
               <div class="eye-icon" v-if="password.length > 0" @click="togglePassword">
```

Comment:
Password.length > 0 is repeating. Can you use computed instead?

Reply:
done